src/Appwrite/Migration/Version/V20.php

```diff
@@ -47,14 +47,17 @@ public function execute(): void
         Console::info('Migrating Collections');
         $this->migrateCollections();
 
-        Console::info('Migrating Functions');
-        $this->migrateFunctions();
+        // No need to migrate stats for console
+        if ($this->project->getInternalId() !== 'console') {
```

Comment:
We can also move these lines to the console if condition

<img width="650" alt="image" src="https://github.com/appwrite/appwrite/assets/23742426/b64216ba-5715-457c-85b5-5211f1e7f4f7">